src/tasks/package.js

```diff
@@ -29,32 +29,33 @@ const packages = [
 module.exports = (config) => {
   json('package.json')
     .merge({
+      main: 'dist/cjs.js',
+      files: [
+        'dist',
+      ],
       engines: {
         // Some versions are skipped because of known issues, see https://github.com/webpack-contrib/organization/issues/7
         node: `>= ${config.minNode} < 5.0.0 || >= 5.10`,
       },
       scripts: {
-        start: 'yarn run serve:dev src',
-        build: "cross-env NODE_ENV=production babel src -d dist --ignore 'src/**/*.test.js'",
-        'clean:dist': 'del-cli dist',
-        lint: 'eslint --cache src test',
+        start: 'yarn run build -- -w',
+        build: 'cross-env NODE_ENV=production babel src -d dist',
+        'clean': 'del-cli dist test/builds',
+        lint: 'eslint --cache src',
         'lint-staged': 'lint-staged',
-        prebuild: 'yarn run clean:dist',
+        prebuild: 'yarn run clean',
         prepublish: 'yarn run build',
         release: 'yarn run standard-version',
         security: 'nsp check',
-        'serve:dev': 'nodemon $2 --exec babel-node',
-        test: 'jest',
+        develop: 'nodemon $2 --exec babel-node',
+        pretest: 'yarn run build && node test/webpack.build.js',
```

Comment:
Get Jest core involved. Here, https://github.com/webpack/webpack/issues/4499 . I think they are willing to help with whatever we might need to make it work for webpack projects.